plugins/openai_plugin.py

```diff
@@ -48,6 +62,35 @@ def _parse_args(self, args):
 
         logger.debug("Model name: %s", self.model_name)
 
+        self.api = args.get('api')
+
+        if not self.api:
+            self.api = 'chat' if "/v1/chat/completions" in self.host else 'legacy'
+
+        if self.api not in APIS:
+            logger.error("Invalid api type: %s", self.api)
+
+        # TODO Make this configurable
+        self.request_defaults = dict(
+            temperature = 0.0,
+            seed = 42,
+        )
```

Comment:
Do these defaults seem reasonable?

Reply:
Yes, though you changed the seed from 10 to 42 :)